public/app/features/integrated-alerting/components/AlertRules/AddAlertRuleModal/AddAlertRuleModal.tsx

```diff
@@ -44,7 +44,13 @@ export const AddAlertRuleModal: FC<AddAlertRuleModalProps> = ({ isVisible, setVi
     try {
       const [channelsListResponse, templatesListResponse] = await Promise.all([
         NotificationChannelService.list(),
-        AlertRuleTemplateService.list(),
+        // Asking for only 100 might be considered a limitation
+        AlertRuleTemplateService.list({
+          page_params: {
+            index: 0,
+            page_size: 100,
```

Comment:
Shouldn't we let the user pick the page size? Maybe 100 could be just the default

Reply:
The problem is this code belongs to the "AddAlertRuleModal", where the user picks a template from the dropdown, which means they don't get to choose the size  :S